backend/compact-connect/lambdas/cloudfront-csp/test/index.test.js

```diff
@@ -0,0 +1,388 @@
+            expect(result.headers['x-frame-options'][0].value).to.equal('DENY');
+            expect(result.headers['x-xss-protection'][0].key).to.equal('X-Xss-Protection');
+            expect(result.headers['x-xss-protection'][0].value).to.equal('1; mode=block');
+            expect(result.headers['referrer-policy'][0].key).to.equal('Referrer-Policy');
+            expect(result.headers['referrer-policy'][0].value).to.equal('strict-origin-when-cross-origin');
+            expect(result.headers['content-security-policy'][0].key).to.equal('Content-Security-Policy');
+            expect(result.headers['content-security-policy'][0].value).to.equal(buildCspHeaders(environment));
+            expect(result.headers.server[0].key).to.equal('Server');
+            expect(result.headers.server[0].value).to.equal('CompactConnect');
+        });
+        it('should successfully return the security headers for ia justin', async () => {
+            const environment = environments.ia.justin;
+            const request = {
+                headers: {
+                    host: [{
+                        value: environment.webFrontend,
+                    }],
+                },
+            };
+            const response = {
+                headers: {},
+            };
+            const config = lambdaConfig({
+                lambdaPath: `index.js`,
+                request,
+                response,
+            });
+            const result = await runLambda(config);
+
+            // console.log(JSON.stringify(result, null, 2)); // @DEBUG
+
+            expect(result.headers['strict-transport-security'][0].key).to.equal('Strict-Transport-Security');
+            expect(result.headers['strict-transport-security'][0].value).to.equal('max-age=31536000; includeSubdomains; preload');
+            expect(result.headers['x-content-type-options'][0].key).to.equal('X-Content-Type-Options');
+            expect(result.headers['x-content-type-options'][0].value).to.equal('nosniff');
+            expect(result.headers['x-frame-options'][0].key).to.equal('X-Frame-Options');
+            expect(result.headers['x-frame-options'][0].value).to.equal('DENY');
+            expect(result.headers['x-xss-protection'][0].key).to.equal('X-Xss-Protection');
+            expect(result.headers['x-xss-protection'][0].value).to.equal('1; mode=block');
+            expect(result.headers['referrer-policy'][0].key).to.equal('Referrer-Policy');
+            expect(result.headers['referrer-policy'][0].value).to.equal('strict-origin-when-cross-origin');
+            expect(result.headers['content-security-policy'][0].key).to.equal('Content-Security-Policy');
+            expect(result.headers['content-security-policy'][0].value).to.equal(buildCspHeaders(environment));
+            expect(result.headers.server[0].key).to.equal('Server');
+            expect(result.headers.server[0].value).to.equal('CompactConnect');
+        });
```

Comment:
I know this was a last discussion on last PR... but I think this might cross the line of being explicit on tests. I _expect_ that these tests are all identical, but I really can't tell easily. Could we refactor? It seems the only thing changing is the environment variable.

Reply:
Factored this to a helper function.

Reply:
Thanks!